std/utf.d

```diff
@@ -306,6 +307,54 @@ pure nothrow @safe @nogc unittest
     });
 }
 
+/**
+Checks if a single character forms a valid code point.
```

Comment:
This here already suggests to me that the name of the function should be `isValidCodepoint`.